db/migrations/20190715134956-create-comment.js

```diff
@@ -0,0 +1,42 @@
+
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Comments', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
+    },
+    userId: {
+      type: Sequelize.INTEGER,
+      required: true,
+      references: {
+        model: 'Users',
+        key: 'id',
+        as: 'user'
+      }
+    },
+    articleId: {
+      type: Sequelize.INTEGER,
+      required: true,
+      references: {
+        model: 'Articles',
+        key: 'id',
+        as: 'article'
+      }
+    },
+    content: {
+      allowNull: false,
+      type: Sequelize.STRING
```

Comment:
Nice PR, but I think you should use Sequelize.TEXT instead of Sequelize.STRING

Reply:
Good catch Frank, thank you for noticing, I found out while I debugging as well.